content/docs/user-guide/components/reference/_index.md

```diff
@@ -146,12 +146,12 @@ The following components are grouped by type as they appear in the O3DE Editor.
 | [Slope Gradient](./gradients/slope-gradient) | Generates a gradient based on the surface angle. |
 | [Surface Mask Gradient](./gradients/surface-mask-gradient) | Generates a gradient based on the underlying surface types. |
 
-<!-- 
-### Networking
+### Multiplayer
 
 | Component | Description | 
 | - | - |
--->
+| [Simple Player Spawner](./multiplayer/simple-player-spawner) | A basic implementation for handling player join and player leave events in a network multiplayer session. |
```

Comment:
From the screenshot, it looks like it's called **Simple Network Player Spawner**. Did you mean to simplify the name? Could be a minor source of confusion.

Reply:
Didn't mean to simplify - thanks for catching!
```suggestion
| [Simple Network Player Spawner](./multiplayer/simple-player-spawner) | A basic implementation for handling player join and player leave events in a network multiplayer session. |
```